tests/test_mle.py

```diff
@@ -177,6 +177,46 @@ def test_infer_node_dgs_w_error():
         assert res.rvalue > 0.9
 
 
+def test_infer_node_vals_and_errs_networkx():
+    edge_noise_stddev = np.random.rand()
+    g = generate_random_valid_regular_graph()
+    g = nx.convert_node_labels_to_integers(g)
+    n_nodes = g.number_of_nodes()
+
+    node_vals, edge_idxs, obs_edge_diffs, edge_stddevs = generate_instance(g, edge_noise_stddev)
```

Comment:
This looks like it only tests the case where all nodes/edges have values. Should strengthen test to verify that if some nodes/edges don't have values, this still behaves as expected.

Reply:
Good call. Currently this does not support the case where some edges are missing values, but I think it can be generalized in tandem with implementing https://github.com/proteneer/timemachine/pull/987#discussion_r1154493745. Will make this change and strengthen the test.

Reply:
Added test for missing edge values in https://github.com/proteneer/timemachine/pull/987/commits/dbeafa92ec3a7313e1b6265fb8aa33d33272f37c and implementation in https://github.com/proteneer/timemachine/pull/987/commits/be601a2ce5e5db904e90eb81b9c5e23992946e77